src/main/java/org/folio/processor/RuleProcessor.java

```diff
@@ -150,12 +156,13 @@ private void applyTranslation(ListValue listValue, ReferenceDataWrapper referenc
       for (int currentIndex = 0; currentIndex < listValue.getValue().size(); currentIndex++) {
       StringValue stringValue = listValue.getValue().get(currentIndex);
       String readValue = stringValue.getValue();
-      RecordInfo recordInfo = stringValue.getRecordInfo();
         try {
           TranslationFunction translationFunction = translationHolder.lookup(translation.getFunction());
           String translatedValue = translationFunction.apply(readValue, currentIndex, translation, referenceData, metadata);
           stringValue.setValue(translatedValue);
         } catch (Exception e) {
+          log.warn("Error while applying translation: function= {}; stringValue= {}; error msg= {}.", translation.getFunction(), stringValue, e.getMessage());
```

Comment:
If we catch exception then log.error() should be here not log.warn.

Reply:
@khandramai 
FYI
The task is based on the document https://folio-org.atlassian.net/wiki/spaces/DD/pages/1779260/Logging.
according to p 1.4 in case of an expected (handled) exception, there must be a WARN-level log message.

please just let me know if I still need to change it to ERROR-level

Reply:
done